pandas/tests/apply/test_invalid_arg.py

```diff
@@ -66,6 +66,15 @@ def test_map_with_invalid_na_action_raises():
         s.map(lambda x: x, na_action="____")
 
 
+def test_map_arg_is_dict_with_invalid_na_action_raises():
+    # https://github.com/pandas-dev/pandas/issues/46588
+    s = Series([1, 2, 3])
+    na_action = "____"
```

Comment:
Could you use `pytest.mark.parameterize` to test `"____"` and `True` (like in the original issue)?

Reply:
Yeah definitely, it's done now.